_applications/equoid.adoc

```diff
@@ -0,0 +1,219 @@
+= Scalable trending item detection on streaming data
+:page-project-name: Equoid
+:page-link: equoid
+:page-liquid:
+:page-weight: 0 
```

Comment:
the `page-weight` should be set to 100, that is the default for all articles. this is mainly here so we can push articles to the top of the list.

Reply:
ACK, added to next rev